src/main/java/Duke.java

```diff
@@ -1,10 +1,186 @@
+            if (userSplitInputs[0].equals("bye")) {
+                sayBye();
+            } else if (userSplitInputs[0].equals("list")) {
+                listItems(todoList);
+            } else if (userSplitInputs[0].equals("done")) {
+                errorCheckingForMarkTasks("done", userSplitInputs,
+                        todoList);
+            } else if (userSplitInputs[0].equals("undone")) {
+                errorCheckingForMarkTasks("undone", userSplitInputs,
+                        todoList);
+            } else {
+                add(todoList, userCommand);
+            }
+
+            // bottom dotted line
+            System.out.println(DOTTED_LINE + '\n');
+
+        } while (!userCommand.equals("bye"));
+
+    }
+
+
+    /**
+     * prints the logo of the chatBot
+     *
+     * @param TAB_SIZE white spaces in front of the printed text for aesthetic purposes
+     */
+    public static void printLogo(final String TAB_SIZE) {
+
+        String logo = "      ____     _    ____\n"
+                + TAB_SIZE + "|  _ \\ __| |__|  _ \\\n"
+                + TAB_SIZE + "| | //|__   __| | //\n"
+                + TAB_SIZE + "| |_\\\\   |  |_| |_\\\\\n"
+                + TAB_SIZE + "|____/   |___/|____/\n";
+
+        System.out.println(logo);
+    }
+
+    /**
+     * greets the user
+     */
+    public static void greetUser(final String TAB_SIZE) {
+
+        System.out.println(TAB_SIZE + "Hello! I'm Bob the Bot, aka BtB.");
+        System.out.println(TAB_SIZE + "Please enter some tasks in the todo list.");
+    }
+
+    /**
+     * Say bye to the user
+     */
+    public static void sayBye() {
+
+        System.out.println(TAB_SIZE + "Bye. Hope to see you again sooooooon (┬┬﹏┬┬)!");
+    }
+
+    /**
+     * add items to an ArrayList
+     *
+     * @param todoList the ArrayList that stores the task added
+     * @param taskDescription the item to be added to todoList
+     */
+    public static void add(ArrayList<Task> todoList,
+                           String taskDescription) {
+
+        Task task = new Task(taskDescription);
+        todoList.add(task);
+
+        System.out.println(TAB_SIZE + "You have added [" + taskDescription +
+                "] to your todo list");
+    }
+
+    /**
+     * lists down the items in the ArrayList and print
+     * them to the terminal
+     *
+     * @param todoList the ArrayLists that contains the stored items
+     */
+    public static void listItems(ArrayList<Task> todoList) {
+
+        if (todoList.size() == 0) {
+            System.out.println(TAB_SIZE + "Your todo list is current void of task," +
+                    " please add some tasks.");
+        } else {
+            System.out.println(TAB_SIZE + "TODO LIST:");
+        }
+
+        for (int i = 1; i <= todoList.size(); i++) {
+
+            System.out.println(TAB_SIZE + TAB_SIZE +
+                    i + "." + todoList.get(i-1).getStatusIcon() +
+                    todoList.get(i-1).getDescription());
```

Comment:
maybe can revise your spacing here
change to
```suggestion
                    todoList.get(i - 1).getDescription());
```